web-common/src/metrics/ErrorEventHandler.ts

```diff
@@ -133,6 +142,12 @@ export class ErrorEventHandler {
       screenName,
       stack,
       message,
+      dashboardState,
     ]);
   }
+
+  private getDashboardState() {
+    // TODO: we need to also save other params that get added after
```

Comment:
What about just storing the whole URL? The main goal is to make it easier to reproduce bugs, so it'd be great if we can just copy/paste the URL into our browser.

Reply:
Ya i thought we save org & project. I guess not, so it makes sense to do this